cli/main.rs

```diff
@@ -161,7 +161,17 @@ async fn run_subcommand(flags: Arc<Flags>) -> Result<i32, AnyError> {
     DenoSubcommand::Uninstall(uninstall_flags) => spawn_subcommand(async {
       tools::installer::uninstall(flags, uninstall_flags).await
     }),
-    DenoSubcommand::Lsp => spawn_subcommand(async { lsp::start().await }),
+    DenoSubcommand::Lsp => spawn_subcommand(async {
+      log::warn!(
+        "{} command is intended to be run by text editors and IDEs and shouldn't be run manually.
+
+Visit https://docs.deno.com/runtime/getting_started/setup_your_environment/ for instruction
+how to setup your favorite text editor.
+
+Press Ctrl+C to exit.
+      ", colors::cyan("deno lsp"));
```

Comment:
Should this only be done when stderr is a terminal? I think this message might show up in vscode's output window?

Reply:
I think it's not a big deal? But sure, I can add a check for terminal here.